qcengine/programs/dftd3.py

```diff
@@ -279,8 +299,10 @@ def parse_output(self, outfiles: Dict[str, str], input_model: "AtomicInput") ->
         }
         output_data["extras"]["local_keywords"] = input_model.extras["info"]
         output_data["extras"]["qcvars"] = calcinfo
-
+        if input_model.driver != "gradient":
+            output_data["extras"]["qcvars"]["PAIRWISE DISPERSION CORRECTION ANALYSIS"] = D3pairs
```

Comment:
We need to check this term explicitly, perhaps the norm of the values?

Reply:
yes, that's the focus of this afternoon's session :-)